Vision/Vision.bs

```diff
@@ -143,15 +143,17 @@ Markup Shorthands: markdown yes
 		we also strive to broaden diversity and inclusion for our own participants.
 	<li id=review>
 		**Thorough Review**:
-		We ensure the technical standards of the Web use broad and 
-		consistent horizontal review to ensure fundamental attributes such as
+		We ensure the technical standards of the Web
+		embody fundamental attributes such as
 		accessibility, 
 		internationalization,
 		sustainability,
 		privacy,
 		security,
 		technical soundness,
-		and architectural integrity.
+		and architectural integrity
+		through consistent deep and wide review.
```

Comment:
```suggestion
		through consistent wide and horizontal review.
```

Reply:
What if, actually, we drop the adjectives here (or maybe replace them with "thorough"), and let the earlier part of the sentence stand in for them?

Reply:
Revised suggestion above to include "deep technical analysis".

Reply:
(approval from @fantasai , committing)